app/assets/stylesheets/style_updates.css.scss

```diff
@@ -52,6 +52,7 @@ html {
 
 
 .entire {
+  font-family: 'Tangerine', cursive;
```

Comment:
Properties should be ordered background, background-repeat, background-size, font-family, height, left, position, top, width